app/views/layouts/base.html.erb

```diff
@@ -22,8 +22,7 @@
         content_for?(:title) ? raw("#{yield(:title)} - #{APP_NAME}") : APP_NAME,
       ) %>
 
-
-  <%= backwards_compatible_javascript_tag do %>
+  <%= javascript_tag(nonce: true) do %>
```

Comment:
should we add a rubocop (or ERBlint?) to make sure that we always set `nonce: true` when calling `javascript_tag`?

Reply:
> should we add a rubocop (or ERBlint?) to make sure that we always set `nonce: true` when calling `javascript_tag`?

Perhaps. Though, to my [own review comment below](https://github.com/18F/identity-idp/pull/6391#discussion_r878393018), a nonce is only required if there's content of the script tag which is expected to be executed inline. I imagine it adds some complexity to the lint.